tests/drone/cache_ocis.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+git diff HEAD~ --name-only
+git diff origin/master --name-only
+if git diff origin/master --name-only | grep .drone.env; then
```

Comment:
Instead of checking for changes against master, shouldn't we check if there is a cache hit?